docs/troubleshooting.md

```diff
@@ -35,3 +36,28 @@ using Minishift.
   systemctl start virtlogd
   systemctl enable virtlogd
   ```
+
+
+## XHYVE drivers
+
+- Could not create vmnet interface, permission denied or no entitlement
+
+    The problem is xhyve driver is not able to clean up vmnet when remove a VM.
+
+    Workaround:
+        vmnet.framework seems to have decide the IP based on below files
+
+            /var/db/dhcpd_leases
+            /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist
+
+    So, If you want to reset `minishift` specific IP database, then make sure you remove `minishift` entry section from `dhcpd_leases` file and reboot your system.
+
+            {
+                ip_address=192.168.64.2
+                hw_address=1,2:51:8:22:87:a6
+                identifier=1,2:51:8:22:87:a6
+                lease=0x585e6e70
+                name=minishift
+            }
+
+    If you want to complete reset IP database, please remove both files manually but very **risky**.
```

Comment:
On line 13,  a minor change, servce>service, though this is not a part of your PR.